docs/cli/getting-started.md

```diff
@@ -97,6 +104,22 @@ Retrieve the configuration audit report:
 starboard get configauditreports deployment/nginx -o yaml
 ```
 
+or
+
+```console
+kubectl get configauditreport -o wide \
```

Comment:
We no longer recommend using kubectl get configauditrepors as the naming convention and label selectors might change. In other works labels set is an implementation details. Also for some K8s object, that contain semicolons we cannot use labels and we fallback to annotations. See the code for details.